### ISSUE TITLE
Feature Request: Support Math variant characters

### ISSUE THREAD

**Issue by PhMajerus** [microsoft/cascadia-code#250]
It would be nice to have all mathematical alphanumeric symbols in Cascadia code.
(Details and list of glyphs at https://en.wikipedia.org/wiki/Mathematical_Alphanumeric_Symbols )

These symbols are designed to show styled forms of Latin and Greek letters in mathematical text, but can also be used to provide alternate visual styles without changing the font.

Having them in the console would both enable mathematicians to write scripts and console tools that report information in an easier to read and understand format, and enable everyone to style plain text by taking advantage of things like italic, bold, ... font variants.

![WinTerm MathVariant](https://user-images.githubusercontent.com/25664275/79673421-e8181480-81d9-11ea-9877-c3b1d03e06ec.png)

As can be seen in the screenshot above, some characters are being taken from different fonts, making the text inconsistent.

**Comment by bcdev-com** [comment]
Please be aware using those math variations for faux-styling plain text rather than their intended purpose is a nightmare for accessibility and those using screen readers.  I would hope not to encourage that.

**Comment by DHowett** [comment]
Cat’s out of the bag there. Our font not supporting those glyphs won’t stop people from pretending they’re Latin letters.

**Comment by bcdev-com** [comment]
Yeah, sadly true.  Mostly howling into the void for random readers who find this thread later.  But from a prioritization perspective, perhaps consider adding those glyphs only as important as displaying math properly in the console?

**Comment by PhMajerus** [comment]
The same could be said about nearly every system that has been used to provide some text styling in terminals. I'm pretty sure most screen readers have issues reading this:
![image](https://user-images.githubusercontent.com/25664275/83956191-a01d8180-a85b-11ea-8873-4eae6958c122.png)

Not supporting these glyphs won't prevent CUI apps and scripts from using them, especially when terminals are limited in how CUI apps can control their output style.

Windows being an OS that tries to provide good accessibility, maybe the opposite could be done instead:
Letting all kind of CUI utilities render properly by including the glyphs, as we have no control over those, but including more advanced accessibility in Windows Terminal to reverse these characters to their Latin equivalents when it detects >10 characters of the same math symbols type (or some other heuristic to detect when it's text instead of math) when the text is provided to screen readers.

I'm not sure if Microsoft Active Accessibility API could be used for this, but, for example, rendering the text using the math symbols on screen as "ℌ𝔢𝔩𝔩𝔬 𝔚𝔦𝔫𝔡𝔬𝔴𝔰 𝔗𝔢𝔯𝔪𝔦𝔫𝔞𝔩!" while, at the same time, detecting those groups and having screen readers TTS those as something like "<pause>Hello Windows Terminal, <volume 50%>written as fraktur symbols<pause>" would give Windows Terminal a big differentiator for users who need accessibility.
